tallycash/extension
# 🗺 Show Me the Way: Prefer alchemy provider for sensitive/important rpc method calls

## Why 

For important rpc calls we should prefer the reliable provider we have. 

## Test 
- import seed
- check balances, activities
- change wallet to optimism 
- open uniswap + connect
- swap on optimism - should not see `Inssufficient liquidity` filckering
- swap on every network on uniswap
- go to 1inch
- swap on every network 
- test internal swaps on every network 
- test send on every network

Latest build: [extension-builds-2662](https://github.com/tallycash/extension/suites/9545265656/artifacts/453904954) (as of Mon, 28 Nov 2022 17:41:31 GMT).
